_data/hosting_providers.json

```
@@ -129,6 +129,16 @@
     "reviewed": "2019.5.31",
     "note": ""
   },
+  {
+    "name": "netclusive",
+    "link": "https://netclusive.de/",
+    "category": "full",
+    "tutorial": "",
+    "announcement": "https://www.netclusive.de/faq/content/31/116/de/kann-ich-meine-webseite-ssl-verschluesseln.html",
+    "plan": "",
+    "reviewed": "2020.10.01",
+    "note": "The website is in German. Old plans of Shared Web Hosting with confixx panel have full supported HTTPS support. New plans of Shared Web Hosting with plesk panel enables the user to register new Let's Encrypt certificates once. Plesk take care of renewals."
```

I think you can just say "full HTTPS support" here.
```suggestion
    "note": "The website is in German. Old plans of Shared Web Hosting with confixx panel have full HTTPS support. New plans of Shared Web Hosting with plesk panel enables the user to register new Let's Encrypt certificates once. Plesk take care of renewals."
```